pandas/core/dtypes/concat.py

```diff
@@ -20,7 +21,7 @@
 )
 from pandas.core.dtypes.generic import ABCCategoricalIndex, ABCRangeIndex, ABCSeries
 
-from pandas.core.arrays import ExtensionArray
+from pandas.core.arrays import ExtensionArray, SparseArray
```

Comment:
Might be creating a circular import? https://dev.azure.com/pandas-dev/pandas/_build/results?buildId=36392&view=logs&j=bef1c175-2c1b-51ae-044a-2437c76fc339&t=770e7bb1-09f5-5ebf-b63b-578d2906aac9

Reply:
Yeah, I already pushed a fix. 
I don't really understand why, though, as `SparseArray` is included in the `pandas.core.arrays` init, just as ExtensionArray.